core/src/HIP/Kokkos_HIP_Parallel_Team.hpp

```diff
@@ -587,7 +587,8 @@ class ParallelReduce<CombinedFunctorReducerType,
   using value_type     = typename ReducerType::value_type;
 
  public:
-  using size_type = HIP::size_type;
+  using functor_type = FunctorType;
+  using size_type    = HIP::size_type;
```

Comment:
was this added becuase needed for the graph?

Reply:
yes